src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java

```diff
@@ -67,12 +67,13 @@ public void handle(Message<JsonObject> message) {
     // 1. Get all pieces for poLineId
     getPieces(query, requestContext)
       .onSuccess(listOfPieces ->
-      // 2. Get PoLine for the poLineId which will be used to calculate PoLineReceiptStatus
-      purchaseOrderLineService.getOrderLineById(poLineIdUpdate, requestContext)
-        .map(poLine -> {
-          if (poLine.getReceiptStatus().equals(PoLine.ReceiptStatus.ONGOING)) {
-            promise.complete();
-          } else {
+        // 2. Get PoLine for the poLineId which will be used to calculate PoLineReceiptStatus
+        purchaseOrderLineService.getOrderLineById(poLineIdUpdate, requestContext)
+          .map(poLine -> {
+            if (poLine.getReceiptStatus() == ReceiptStatus.CANCELLED || poLine.getReceiptStatus() == ReceiptStatus.ONGOING) {
```

Comment:
since it is used in 2 places, maybe it makes sense to introduce constant something like this
public static List<ReceiptStatus> STATUSES_TO_SKIP = Arrays.asList(ReceiptStatus.CANCELLED, ReceiptStatus.ONGOING);
Because in some future changes someone can forgot about second place

Reply:
`ReceiptStatusConsistency::calculatePoLineReceiptStatus` and `CheckinReceivePiecesHelper::calculatePoLineReceiptStatus` use `Piece` class from different packages: `org.folio.rest.jaxrs.model`, `org.folio.rest.acq.model`. We can't combine them however we should remember that changes don in one place should be reflected in an another. Yes we can unify status check but maybe we can plan combining whole logics in the future. WDYT?

Reply:
interesting why we use 2 different dtos and not single one, but if we can't refactor it to combine, we can lease as it